ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "ibm.ibmmq"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Since this is an improvement, better to change the version to `1.1.0`?

Reply:
Fixed with [287e145af28774438363fb78be7e6254dea15481](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/49/commits/287e145af28774438363fb78be7e6254dea15481)